ParkenDD/Colors.swift

```diff
@@ -90,4 +104,31 @@ extension UIColor {
 		}
 		self.init(red:red, green:green, blue:blue, alpha:alpha)
 	}
+
+    // MARK: - Properties on UIColor
+    static var highAvailabilityColor = UIColor(rgba: "#006a39")
+    static var mediumAvailabilityColor = UIColor(rgba: "#1daa8c")
+    static var lowAvailabilityColor = UIColor(rgba: "#7f0304")
+    static var noAvailabilityColor = UIColor(rgba: "#5c5c5c")
+
+    /// Source: https://theswiftdev.com/uicolor-best-practices-in-swift/
+    /// Get the RGB components of a UIColor
+    var rgbaComponents: (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) {
```

Comment:
We don't use `rgbaComponents` anywhere. Remov eit.

Reply:
Deleted